Client/Frontend/Home/ActivityStreamPanel.swift

```diff
@@ -342,20 +368,29 @@ extension ActivityStreamPanel {
                 withUserData: userData,
                 toApplication: UIApplication.sharedApplication())
         })
-
+        
+        let removeAction = ActionOverlayTableViewAction(title: NSLocalizedString("Remove", comment: "Label for \"Remove\" button"), iconString: "action_remove", handler: { action in
+            self.topSitesManager.collectionView(collectionView!, deleteItemAtIndexPath: indexPath!)
```

Comment:
Sure, this worked! But, I still had to keep the ASHorizontalScrollCell.collectionView property public because that's the only way to get the indexPath for cells within the collection view.